proofzero/rollupid
# feat(console): App storage - usage section in billing and invoicing

### Description

### Related Issues

- Closes #2821 

### Testing

- Create personal apps and enable storage on them
- Create group apps and enable storage on them
- Disable storage or delete apps
- See changes propagate
